pkgs/development/python-modules/nix-prefetch-github/default.nix

```diff
@@ -9,13 +9,17 @@
 
 buildPythonPackage rec {
   pname = "nix-prefetch-github";
-  version = "2.3.2";
+  version = "2.4";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "18xj618zjs13ib7f996fnl0xiqig0w48yns45nvy3xab55wximdx";
+    sha256 = "sha256-PVB/cL0NVB5pHxRMjg8TLatvIvHjfCvaRWBanVHYT+E=";
```

Comment:
We don't use SRI hashes since they don't work with Nix 2.0 which is the current minimal version to evaluate nixpkgs. See also #89308.